app/src/main/java/org/resolvetosavelives/red/newentry/search/OngoingPatientEntry.kt

```diff
@@ -6,8 +6,11 @@ data class OngoingPatientEntry(
     val phoneNumbers: PhoneNumbers? = null,
     val bloodPressureMeasurements: BloodPressureMeasurement? = null
 ) {
+  fun hasInvalidAgeOrDob(): Boolean {
```

Comment:
Do you think we should rename `hasInvalidAgeOrDob()` to `areBothAgeAndDobEmpty()`?